src/api/emit.js

```diff
@@ -25,5 +30,8 @@ export default function (elem, name, opts = {}) {
   if (opts.cancelable === undefined) {
     opts.cancelable = true;
   }
+  if (opts.composed === undefined) {
```

Comment:
I think it's worth fixing support for the 1.x releases and raising an issue and possibly going 2.0 soonish.